app/poros/github_results.rb

```diff
@@ -0,0 +1,12 @@
+class GithubResults
+  def repos
+    body = GithubService.new.repos_body
+    body.map do |repo_data|
+      Repo.new(repo_data)
```

Comment:
Call the github api service > return the repo body > instantiate Repo objects from the body array

Reply:
nice! Also, excellent explanation utilizing the ">"!